contracts/SnappAuction.sol

```diff
@@ -76,17 +152,7 @@ contract SnappAuction is SnappBase {
             auctions[auctionIndex].size == maxUnreservedOrderCount() || 
```

Comment:
How would we ensure the batch isn't maxed out without it. People also shouldn't be able to cancel generic orders.

Reply:
Yeah, not sure what to do here...

Probably, I would keep it in here, to avoid new logic. And the current logic (or similar) 
will have to be reimplemented anyways once we have the fee model.